src/pages/resources/explore-books/explore-books.ts

```diff
@@ -0,0 +1,198 @@
+import {Component} from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import {Map} from "@app/app";
+import {Environment, InteractSubtype, InteractType, PageId} from "@app/service/telemetry-constants";
+import {ContentSearchCriteria, ContentService, SearchType} from "sunbird-sdk/dist";
+import {AppHeaderService, CommonUtilService} from "@app/service";
+import { animate, group, state, style, transition, trigger } from '@angular/animations';
+
+
+/**
+ * Generated class for the ExploreBooksPage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@IonicPage()
+@Component({
+  selector: 'page-explore-books',
+  templateUrl: 'explore-books.html',
+  animations: [
+    trigger('appear', [
+      state('true', style({
+        left: '{{left_indent}}',
+      }), { params: { left_indent: 0 } }), // default parameters values required
+
+      transition('* => classAnimate', [
+        style({ width: 5, opacity: 0 }),
+        group([
+          animate('0.3s 0.2s ease', style({
+            transform: 'translateX(0) scale(1.2)', width: '*',
+          })),
+          animate('0.2s ease', style({
+            opacity: 1
+          }))
+        ])
+      ]),
+    ]),
+    trigger('ScrollHorizontal', [
+      state('true', style({
+        left: '{{left_indent}}',
+        transform: 'translateX(-100px)',
+      }), { params: { left_indent: 0 } }), // default parameters values required
+
+      transition('* => classAnimate', [
+        // style({ width: 5, transform: 'translateX(-100px)', opacity: 0 }),
+        group([
+          animate('0.3s 0.5s ease', style({
+            transform: 'translateX(-100px)'
+          })),
+          animate('0.3s ease', style({
+            opacity: 1
+          }))
+        ])
+      ]),
+    ])
+  ]
+
+})
+export class ExploreBooksPage {
+
+  categoryGradeLevels: any;
+  subjects: any;
+  mimeTypes = [
```

Comment:
Hard Coded Names not localized ideally should be read from localization file of english